packages/pagedown/test.R

```diff
@@ -0,0 +1,19 @@
+options(download.file.method = "curl")
+install.packages("pagedown", repos = "https://cran.rstudio.com", type = "source")
+
+# Test against Chromium installation
+pagedown::find_chrome()
```

Comment:
I think this is sufficient to test that the package installed properly and system dependencies are installed, once `pandoc` is available from the docker image and `websocket` is responsible for `libssl`.

Reply:
https://github.com/rstudio/shinyapps-package-dependencies/pull/220/commits/3bdedd10a4dcfc2d1dcc417ce25397a43de2cb0c